src/o365/graph/commands/schemaextension/schemaextension-list.ts

```diff
@@ -0,0 +1,156 @@
+import config from '../../../../config';
+import commands from '../../commands';
+import request from '../../../../request';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import GraphCommand from '../../../base/GraphCommand';
+import Utils from '../../../../Utils';
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  status?: string;
+  owner?: string;
+  pageNumber?: string;
+  pageSize?: string;
+}
+
+class GraphSchemaExtensionListCommand extends GraphCommand {
+  public get name(): string {
+    return `${commands.SCHEMAEXTENSION_LIST}`;
+  }
+
+  public get description(): string {
+    return 'Get a list of schemaExtension objects created in the current tenant, that can be InDevelopment, Available, or Deprecated.';
+  }
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.status = typeof args.options.status !== 'undefined';
+    telemetryProps.owner = typeof args.options.owner !== 'undefined';
+    telemetryProps.pageNumber = typeof args.options.pageNumber !== 'undefined';
+    telemetryProps.pageSize = typeof args.options.pageSize !== 'undefined';
+    return telemetryProps;
+  }
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: () => void): void {
+    const filter: string = this.getFilter(args.options);
+    let url = `${this.resource}/v1.0/schemaExtensions?$select=*${(filter.length > 0 ? '&' + filter : '')}`;
+   
+      if (args.options.pageNumber && Number(args.options.pageNumber) > 0) {
+        const rowLimit: string = `&$top=${Number(args.options.pageSize ? args.options.pageSize : 10) * Number(args.options.pageNumber+1)}`;
+        url += rowLimit;
+      }
+     const requestOptions: any = {
+      url: url,
+      headers: {
+        accept: 'application/json;odata.metadata=none',
+        'content-type': 'application/json'
+      },
+      json: true
+    };
+    request.get(requestOptions)
+      .then((res: any): void => {
+       if (res.value && res.value.length > 0) {
+        const size=args.options.pageSize?parseInt(args.options.pageSize):parseInt(res.value.length);
+        cmd.log(res.value.slice(-size));
```

Comment:
I have changed this line to 
```
if(args.options.output !== 'json' && result.length > 1) {
            cmd.log(result.map((x: any) => ({
              id: x.id, 
              description: x.description,
              targetTypes: x.targetTypes,
              status: x.status,
              owner: x.owner,
              properties: JSON.stringify(x.properties)
            })));
          } else {
            cmd.log(result);
          }
```

The issue was that the table results was returning [Object] instead of json text when more than 2 schema extensions.
![image](https://user-images.githubusercontent.com/1049476/85933384-5827d280-b8ce-11ea-93de-6a35d790d153.png)

Reply:
Oh nice 🙌🏼 Thanks Velin !